### PR TITLE
hyperion-ng: init at 2.0.12

### PR DESCRIPTION
**Pull request by Algram**
###### Description of changes

I've added the `hyperion.ng` package as requested here: https://github.com/NixOS/nixpkgs/issues/108333

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -519,6 +519,12 @@
     githubId = 38869148;
     name = "Alex Eyre";
   };
+  algram = {
+    email = "aliasgram@gmail.com";
+    github = "Algram";
+    githubId = 5053729;
+    name = "Alias Gram";
+  };
```
Comment (bobby285271) [inline maintainers/maintainer-list.nix:527]: Please put this in a separate commit, so there are exactly 2 commits in this PR.

The commit message should be:
- `maintainers: add algram` - for the `maintainers/maintainer-list.nix` changes
- `hyperion-ng: init at 2.0.12` - for other changes

File: pkgs/applications/video/hyperion-ng/default.nix [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  version = "2.0.12";
+
+  src = fetchFromGitHub {
+    owner = "hyperion-project";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-J31QaWwGNhIpnZmWN9lZEI6fC0VheY5X8fGchQqtAlQ=";
+  };
+
+  buildInputs = [ libxcb qtbase qtsvg python3 qtserialport mbedtls libcec flatbuffers protobuf libusb1 hidapi libXrandr qtx11extras libX11 avahi-compat wrapQtAppsHook];
```
Comment (bobby285271) [inline pkgs/applications/video/hyperion-ng/default.nix:34]: `wrapQtAppsHook` should go to `nativeBuildInputs`

(btw I actually prefer one package per line, also will be nice to format the file with nixpkgs-fmt)

**Comment by Algram** [comment]
@bobby285271 Thanks for the review! I've made the changes you requested and formatted everything with `nixfmt`.
